app/src/main/java/com/android/sample/model/profile/ProfilesRepositoryFirestore.kt

```diff
@@ -31,13 +35,18 @@ open class ProfilesRepositoryFirestore @Inject constructor(private val db: Fireb
   private fun documentToUser(document: DocumentSnapshot): User? {
     return try {
       val id = document.id
```

Comment:
if no more debugging is needed, logs could be removed and reused when needed.

Reply:
I'm totally on phase with what @DAFERMOH  has said